django_project/certification/templates/course/detail.html

```diff
@@ -136,6 +136,12 @@ <h4 class="panel-heading" style="padding-left:10px; padding-right: 10px">
             data-title="Download all issued certificates from this course">
             <span class="glyphicon glyphicon-download-alt"></span>
             </a>
+        <a id='upload-attendee' class="btn btn-default btn-sm tooltip-toggle"
```

Comment:
Can you put it next to the other course attendee icon?

Reply:
Alright @timlinux.

Reply:
>Can you put it next to the other course attendee icon?

Updated.